CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 Documentation for rocRAND is available at
 [https://rocm.docs.amd.com/projects/rocRAND/en/latest/](https://rocm.docs.amd.com/projects/rocRAND/en/latest/)
 
+
+## (Unreleased) rocRAND 3.2.0 for ROCm 6.4
+
+### Added
+* Added regression tests to `rtest.py`. Regression tests are a subset of tests that caused hardware problems for past emulation environments.
```

Comment:
This sentence is a bit confusing. Did the regression tests cause the hardware problems? Or do they validate scenarios that have caused problems in the past? Maybe if you describe the situation, we can reword this so it's clearer.

Reply:
Hi, they are the ones that caused hardware problems in the past.

Reply:
OK, we can leave it then.